README.md

```diff
@@ -30,76 +41,90 @@ $ helm install ohmyglb ohmyglb/ohmyglb
 See [values.yaml](https://github.com/AbsaOSS/ohmyglb/blob/master/chart/ohmyglb/values.yaml)
 for customization options.
 
-### Local Playground
-
-####  Deploy local cluster
-
-```sh
-$ make deploy-local-cluster
+### Local Playground Install
+
+#### Environment prerequisites
+
+ - install **GO 1.14**
+ 
+ - install **GIT**     
+    - ensure SSH key is set for your account; see https://github.com/settings/keys  
+    
+ - install **gnu-sed** if you don't have it  
+    - update $PATH in `~/.bash_profile` by sed path `export PATH=/usr/local/opt/gnu-sed/libexec/gnubin:$PATH` 
+   
+ - install **Docker**
+    - ensure you are able to push/pull from your docker registry
+    - to run multiple clusters reserve 8GB of memory  
+    
+      ![](docs/images/docker_settings.png)
+      <div>
+        <sup>above screenshot is for <strong>Docker for Mac</strong> and that options for other Docker distributions may vary</sup>
+      </div>
```

Comment:
why it is html markup here out of the sudden ?

Reply:
it's note description under picture 
https://github.com/AbsaOSS/ohmyglb/blob/1_HowTo_documentation/README.md

Reply:
ok, got it, fancy  :)

Reply:
oh yes, thats from github doc you posted https://github.blog/2020-04-09-github-protips-tips-tricks-hacks-and-secrets-from-lee-reilly/